### ISSUE TITLE
Possibly a mistake in the documentation (Calling the JS Function with EvaluateScript())

### ISSUE THREAD

**Issue by mat-r** [ultralight-ux/Ultralight#337]
[At this place](https://docs.ultralig.ht/docs/calling-a-js-function-from-c#calling-the-js-function-with-evaluatescript), the code snippet is
```c
void MyApp::OnDOMReady(View* caller,
                       uint64_t frame_id,
                       bool is_main_frame,
                       const String& url) {
  ///
  /// Call ShowMessage() by evaluating a string of JavaScript
  ///
  view->EvaluateScript("ShowMessage('Howdy!')");
  
}
```
Shouldn't it be caller instead of view?
```c
  caller->EvaluateScript("ShowMessage('Howdy!')");
```

**Comment by Superxwolf** [comment]
Yep, I believe the documentation has a button to suggesting typos correction directly on the doc page.

**Comment by Mat-R** [comment]
Indeed, however it looks like a readme account is required. Is it the only way to report typos?

**Comment by adamjs** [comment]
Thanks! This should be fixed in latest docs for 1.3.0.
